content/it/bare-metal-machine.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Macchina Bare Metal
+status: Completed
+category: tecnologia
+tags: ["infrastruttura", "", ""]
+---
+
+## Cos'è
+
+Bare metal (letteralmente _nudo metallo_) si riferisce a un computer fisico, più specificamente un server, che ha uno e un solo sistema operativo.
+Nell'informatica moderna questa distinzione è importante perché molti, se non la maggior parte, dei server sono [macchine virtuali](/it/virtual-machine/).
+Un server fisico è tipicamente un computer di grandi dimensioni con un hardware potente.
+Installare un sistema operativo ed eseguire le applicazioni direttamente su quell'hardware fisico senza [virtualizzazione](/it/virtualization/), si definisce eseguire su “bare metal.”
+
+## Quali problematiche affronta
+
+Accoppiare un sistema operativo con un computer fisico è il modello originale dell'informatica.
+Tutte le risorse del computer fisico sono direttamente disponibili per il sistema operativo, e senza livelli di virtualizzazione non c'è ritardo artificiale nella traduzione delle istruzione del sistema operativo all'hardware.
+
+## In che modo aiuta
+
+Dedicando tutte le risorse computazionali di un computer a un singolo sistema operativo si fornisce potenzialmente la miglior performance possibile al sistema operativo.
+Se è necessario eseguire un carico di lavoro che deve avere un accesso estremamente veloce alle risorse hardware, allora il bare metal può essere la giusta soluzione.
+
+Nel contesto di [applicazioni cloud native](/it/cloud-native-apps/),
+generalmente si pensa alle performance in termini di [scalabilità](/it/scalability/) a un numero più alto di eventi concomitanti,
+che possono essere gestiti [scalando orizzontalmente](/it/horizontal-scaling/) (aggiungendo più macchine).
```

Comment:
```suggestion
che possono essere gestiti utilizzando la proprietà di [scalabilità orizzontale](/it/horizontal-scaling/) (aggiungendo più macchine al pool di risorse).
```
(ne ho fatto una battaglia personale:) il verbo "scalare" non ha significato proprio di "to scale" in italiano; finora abbiamo usato sempre delle locuzioni.